Notebooks/Spark.NET C#/GitHub Projects Advanced.ipynb

```diff
@@ -0,0 +1,576 @@
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": "+---+--------------------+---------+------------+----------+-------------------+\n| id|                 sha|author_id|committer_id|project_id|         created_at|\n+---+--------------------+---------+------------+----------+-------------------+\n|  2|397238b49c88d1d8e...|        2|           2|         1|2012-08-01 13:25:36|\n|  3|55bf5367875ec9e81...|        2|           2|         1|2012-06-18 03:39:30|\n|  4|9d653ea84c6df1b90...|        2|           2|         1|2012-06-11 07:47:16|\n|  5|fd00ce155f1a9842d...|        2|           2|         1|2012-06-11 07:45:07|\n|  6|641b94c68ecd63478...|        2|           2|         1|2012-05-07 06:00:56|\n|  7|3f7d2c8f8dd589222...|        2|           2|         1|2012-03-08 04:47:19|\n|  8|8adee7b7a7d634630...|        2|           2|         1|2012-03-08 04:40:43|\n|  9|ecb30132a2d978a70...|        2|           2|         1|2012-03-08 04:40:25|\n| 10|c1d057e040786c909...|        2|           2|         1|2012-03-08 04:24:22|\n| 11|eefadec57441a172d...|        2|           2|         1|2012-03-07 22:06:35|\n| 12|92c2802570f553f49...|        2|           2|         1|2012-02-18 02:11:50|\n| 13|3d4dcd08e1e14c62e...|        2|           2|         1|2012-02-18 02:09:27|\n| 14|6829cd56d776b7926...|        2|           2|         1|2012-02-13 18:43:47|\n| 15|b8bf2fdbd482f3c8e...|        2|           2|         1|2012-01-17 02:39:20|\n| 16|94c52c310f8850503...|        2|           2|         1|2012-01-17 00:03:04|\n| 17|55e5630f630dd93f9...|        2|           2|         1|2012-01-16 12:07:50|\n| 18|f060b5c9f596c0fea...|        2|           2|         1|2012-01-16 12:02:10|\n| 19|bcfb0beee5704c32d...|        2|           2|         1|2012-01-16 11:56:52|\n| 20|8c8f85cbbc367f69b...|        2|           2|         1|2012-01-04 10:18:30|\n| 21|def4122b4ec68c1aa...|        2|           2|         1|2012-01-04 10:53:37|\n+---+--------------------+---------+------------+----------+-------------------+\nonly showing top 20 rows"
+          },
+          "execution_count": 3,
+          "metadata": {}
+        }
+      ],
+      "metadata": {
+        "cell_status": {
+          "execute_time": {
+            "duration": 1271.4638671875,
+            "end_time": 1571951706712.9
+          }
+        },
+        "collapsed": false
+      },
+      "source": [
+        "DataFrame commits = spark\n",
+        "            .Read()\n",
+        "            .Option(\"header\", \"true\")\n",
+        "            .Schema(\"id INT, sha STRING, author_id INT, committer_id INT, \" +\n",
+        "                    \"project_id INT, created_at TIMESTAMP\")\n",
+        "            .Csv(\"<path_to_ghtorrent/commits.csv>\");\n",
```

Comment:
We're assuming they're going to provide `<path_to_ghtorrent/commits.csv>`?

Reply:
Please see my comment on line 71. Open to ideas on how to make this better